### DISCUSSION TITLE
Is using hooks the only way to create an animation state?

### DISCUSSION THREAD

**Discussion by HyopeR** [software-mansion/react-native-reanimated#3613] [Q&A]
Hello. I am working with the Reanimated package. 
This is how we used to create animation states in older versions.

```
new Animated.Value(0)
```

After the Reanimated 2 update, the animation state is created as follows.
```
useSharedValue(0);
```

My question is exactly **can I create animation state without using hook useSharedValue() ?**
I want to manage many animations by writing an external service, but creating a state with hook only is a big constraint.
Can't new Animated.Value() be done as before?

**Comment by HyopeR** [comment]
After examining the source code a bit, I found that I can use these two functions. **makeMutable** and **cancelAnimation**. Is it safe to use these two instead of hooks?

**Reply by tomekzaw** [reply]
Hey, these functions are meant to be used internally. So the answer to your question is that `useSharedValue` is the way to go.
